src/content/guides/code-splitting.md

```diff
@@ -294,6 +302,49 @@ __src/index.js__
 ```
 
 
+## Prefecting/Preloading modules
+
+webpack 4.6.0+ adds support for prefetching and preloading.
+
+Using these inline directives while declaring your imports allows webpack to output “Resource Hint” which tells the browser that this resource is probably (prefetch) or definitely (preload) needed for some navigation in the future.
```

Comment:
Actually preload is indicating the resource might be needed during the current navigation and prefetch for future navigation, not about how likely it will be needed. Can we change this part to rflect that?